BeeSwift/GoalViewController.swift

```diff
@@ -41,6 +43,14 @@ class GoalViewController: UIViewController, UITableViewDelegate, UITableViewData
         }
         self.title = self.goal.slug
         
+        if #available(iOS 12.0, *) {
+            let viewGoalActivity = self.viewGoalShortcut()
+            self.userActivity = viewGoalActivity
```

Comment:
Supporting iOS Search via NSUserActivity has been around since iOS 8. It looks like this is meant for use inside the Shortcuts app.


Beeminder "add data" available within Shortcuts:
![Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-30 at 10 08 53](https://user-images.githubusercontent.com/4656811/112955873-2800af00-9140-11eb-8c89-e563df106349.png)
![Simulator Screen Shot - iPhone 12 Pro Max - 2021-03-30 at 10 09 04](https://user-images.githubusercontent.com/4656811/112955879-28994580-9140-11eb-91ae-e359024e1ac7.png)

Reply:
Having the data pop up in search is not that far off. 

--




some screenshots from using NSUserActivity to integrate with iOS Search, from Aug 2020:
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-08-06 at 17 25 12](https://user-images.githubusercontent.com/4656811/112956375-af4e2280-9140-11eb-9941-ae951e40ebcd.png)

![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-08-06 at 14 31 25](https://user-images.githubusercontent.com/4656811/112956363-acebc880-9140-11eb-93ba-349eccad443f.png)